File: backend/Models/DTO/NotificationDto.cs
namespace TestScriptTracker.Models.DTO
{
    public class NotificationDto
    {
        public Guid NotificationId { get; set; }
        public string NotificationTitle { get; set; } = string.Empty;
        public string Message { get; set; } = string.Empty;
        public DateTime NotificationDate { get; set; }
        public int NotificationTypeId { get; set; }
        public Guid? ProjectId { get; set; }
       
        //for the user:
        public Guid Id { get; set; }

        public string SenderName { get; set; }

        public string SenderSurname { get; set;  }


        public bool IsOpened { get; set; }
    }
}

File: frontend/src/app/models/notificationDTO.ts
export interface NotificationDto {
  notificationId: string;
  notificationTitle: string;
  message: string;
  notificationDate: Date;
  notificationTypeID: number;
  projectID: number | null;
  Id: string;
  senderName: string; 
  senderSurname: string; 
  isOpened: boolean;
}
